DataFormats/CSCDigi/interface/CSCCorrelatedLCTDigi.h

```diff
@@ -133,6 +141,14 @@ class CSCCorrelatedLCTDigi {
   /// set cscID
   void setCSCID(unsigned int c) { cscID = c; }
 
+  /// set high-multiplicity bits
+  void setHMT(const unsigned int h) { hmt = h; }
+
+  /// Distinguish Run-1/2 from Run-3
+  bool isRun3() const { return version_ == Version::Run3; }
+
+  void setRun3(bool isRun3) { version_ = Version::Run3; }
```

Comment:
What does the bool "isRun3" do here?  It looks like "version_" is set to Run3 whether "isRun3" is true or false.

Reply:
yeah, sorry about that, that come from my hasty suggestion...